From 5525076f2ef4ec59adc3b9259dc2e780c4430c4e Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Wed, 24 Jul 2024 09:14:29 +0800
Subject: [PATCH 10/11] Revert MoveEmitter-mips64.cpp part in "Bug 1870747: Use
 ABIType instead of MoveOp::Type for callWithABI r=rhunt"

This reverts part of commit 1e5b0f3713070c2d43a548ae0f9e3c97e64004f7.
---
 js/src/jit/mips64/MoveEmitter-mips64.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/js/src/jit/mips64/MoveEmitter-mips64.cpp b/js/src/jit/mips64/MoveEmitter-mips64.cpp
index 53ee23820a..70217a37f8 100644
--- a/js/src/jit/mips64/MoveEmitter-mips64.cpp
+++ b/js/src/jit/mips64/MoveEmitter-mips64.cpp
@@ -30,7 +30,7 @@ void MoveEmitterMIPS64::breakCycle(const MoveOperand& from,
         masm.storeFloat32(to.floatReg(), cycleSlot(slotId));
       }
       break;
-    case ABIType::Float64:
+    case MoveOp::DOUBLE:
       if (to.isMemory()) {
         FloatRegister temp = ScratchDoubleReg;
         masm.loadDouble(getAdjustedAddress(to), temp);
@@ -85,7 +85,7 @@ void MoveEmitterMIPS64::completeCycle(const MoveOperand& from,
         masm.loadFloat32(cycleSlot(slotId), to.floatReg());
       }
       break;
-    case ABIType::Float64:
+    case MoveOp::DOUBLE:
       if (to.isMemory()) {
         FloatRegister temp = ScratchDoubleReg;
         masm.loadDouble(cycleSlot(slotId), temp);
-- 
2.46.0

